pkg/providers/instance/instance.go

```diff
@@ -136,15 +135,11 @@ func (p *Provider) Get(ctx context.Context, id string) (*Instance, error) {
 
 func (p *Provider) List(ctx context.Context) ([]*Instance, error) {
 	var out = &ec2.DescribeInstancesOutput{}
-	tagKeys := []string{v1alpha5.ProvisionerNameLabelKey}
-	if nodepoolutil.EnableNodePools {
-		tagKeys = append(tagKeys, corev1beta1.NodePoolLabelKey)
-	}
 	err := p.ec2api.DescribeInstancesPagesWithContext(ctx, &ec2.DescribeInstancesInput{
 		Filters: []*ec2.Filter{
 			{
 				Name:   aws.String("tag-key"),
-				Values: aws.StringSlice(tagKeys),
+				Values: aws.StringSlice([]string{v1alpha5.ProvisionerNameLabelKey, corev1beta1.NodePoolLabelKey}),
```

Comment:
Is this an OR or an AND?

Reply:
This is an OR

Reply:
It's: there must be a value from this set of things